tracer/src/Datadog.Trace.Tools.Runner/Utils.cs

```diff
@@ -509,9 +534,152 @@ internal static bool IsAlpine()
                 envVars["COR_PROFILER_PATH_ARM64"] = tracerProfilerArm64;
             }
 
+            if (!string.IsNullOrEmpty(devPath))
+            {
+                envVars["DEVPATH"] = devPath;
+            }
+
             return envVars;
         }
 
+        private static bool EnsureDatadogTraceIsInTheGac(string tracerHome)
+        {
+            // We try to ensure Datadog.Trace.dll is installed in the gac for compatibility with .NET Framework fusion class loader
+            // Let's find gacutil, because CI Visibility runs with the SDK / CI environments it's probable that's available.
+            Log.Debug("EnsureDatadogTraceIsInTheGac: Looking for gacutil");
+            var cmdResponse = ProcessHelpers.RunCommand(new ProcessHelpers.Command("where", "gacutil"));
+            if (cmdResponse?.ExitCode == 0 &&
+                cmdResponse.Output.Split(["\n", "\r\n"], StringSplitOptions.RemoveEmptyEntries) is { Length: > 0 } outputLines &&
+                outputLines[0] is { Length: > 0 } gacPath)
+            {
+                Log.Debug("EnsureDatadogTraceIsInTheGac: gacutil was found.");
+                var cmdGacListResponse = ProcessHelpers.RunCommand(new ProcessHelpers.Command(gacPath, "/l"));
+                if (cmdGacListResponse?.ExitCode == 0)
```

Comment:
Is this worth having a logging `else` ? The exit code should be zero, right?

Reply:
True, good catch, added in [3e61238](https://github.com/DataDog/dd-trace-dotnet/pull/5129/commits/3e61238eb44292f07e8e37ecb1cdc591b4b5ed2e)